sklearn/neighbors/tests/test_approximate.py

```diff
@@ -229,7 +229,8 @@ def test_radius_neighbors_boundary_handling():
 
     # Build a LSHForest model with hyperparameter values that always guarantee
     # exact results on this toy dataset.
-    lsfh = LSHForest(min_hash_match=0, n_candidates=n_points).fit(X)
+    lsfh = LSHForest(min_hash_match=0, n_candidates=n_points,
+                     random_state=42).fit(X)
```

Comment:
I don't know why this is needed ... but it is needed without it you get an error consistently like [this](https://travis-ci.org/lesteve/scikit-learn/jobs/176360295#L1989) if you run the tests with `nosetests sklearn`. The error can not be reproduced only with running a single test function or the single file with `nosetests sklearn/neighbors/tests/test_approximate.py` and I tried in a loop 100 times. Still trying to get to the bottom of this ...